src/pages/Task.tsx

```diff
@@ -43,6 +43,7 @@ export const Task: React.FC = () => {
     GetTaskQueryVariables
   >(GET_TASK, {
     variables: { taskId: id, execution: selectedExecution },
+    fetchPolicy: "network-only",
```

Comment:
why change this from cache-first ([default option](https://www.apollographql.com/docs/react/data/queries/#supported-fetch-policies)) to network-only?

Reply:
We want to show the status as "known-failure" on patch details, but show the real failure on the task page. If it pulls it from the cache, we can't show the user the real failure since it's overwritten by "known-issue".